node/node.go

```diff
@@ -272,6 +310,43 @@ func (nn *nexNode) initializeSupervisionTree() error {
 	}
 	nn.options.Logger.Debug("Actors started", slog.Any("running", running))
 
+	time.Sleep(250 * time.Millisecond)
```

Comment:
Same question as above.. what is this for?

Reply:
I believe the actor framework author does it in their examples and its an artifact of that.  I haven't tried removing it and seeing if it matters....will look into it